src/NLog.Targets.Syslog/Settings/SdElementConfig.cs

```diff
@@ -13,13 +13,13 @@ namespace NLog.Targets.Syslog.Settings
     /// <summary>Syslog SD-ELEMENT field configuration</summary>
     public class SdElementConfig : NotifyPropertyChanged, IDisposable
     {
-        private SdIdConfig sdId;
+        private Layouts.SimpleLayout sdId;
```

Comment:
NLog 5.0 have broken the implicit conversion operator, so I needed to use SimpleLayout directly.